src/DssSpell.sol

```diff
@@ -48,134 +57,134 @@ contract DssSpellAction is DssAction {
+
+            // ---------- Update PSM-USDC-A DC-IAM ----------
+            // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+            // DC-IAM line: Set to 50M
+            dstMaxLine:   50 * MILLION * RAD,
+            // DC-IAM gap: Set to 20M
+            dstGap:       20 * MILLION * RAD,
+            // DC-IAM ttl: Set to 12h
+            dstTtl:       12 hours,
+
+            // ---------- Set up LITE-PSM-USDC-A DC-IAM ----------
+            // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+            // DC-IAM line: 10B (Unchanged)
+            srcMaxLine:   10 * BILLION * RAD,
+            // DC-IAM gap: Decrease for 20M from 400M to 380M
+            srcGap:       380 * MILLION * RAD,
+            // DC-IAM ttl: 12h (Unchanged)
+            srcTtl:       12 hours,
+
+            // ---------- Initial USDC Migration from PSM-USDC-A to LitePSM ----------
+            // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+            // Migrate 20 million USDC from PSM-USDC-A to LITE-PSM-USDC-A
+            dstWant:      20 * MILLION * WAD,
+            // Leave at least 200M USDC reserves in PSM-USDC-A
+            srcKeep:      200 * MILLION * WAD,
+
+            // ---------- Chainlog additions ----------
+            // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+            // Add 0x467b32b0407Ad764f56304420Cddaa563bDab425 as LITE_PSM_MOM
+            psmMomKey:    "LITE_PSM_MOM",
+            // Add 0xf6e72Db5454dd049d0788e411b06CfAF16853042 as MCD_LITE_PSM_USDC_A
+            dstPsmKey:    "MCD_LITE_PSM_USDC_A",
+            // Add 0x37305B1cD40574E4C5Ce33f8e8306Be057fD7341 as MCD_LITE_PSM_USDC_A_POCKET
+            dstPocketKey: "MCD_LITE_PSM_USDC_A_POCKET"
+        });
+        DssLitePsmMigrationPhase1.initAndMigrate(dss, inst, cfg);
+
+        // ---------- GSM Delay Update ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+        // Decrease the GSM Pause Delay by 14h, from 30h to 16h
+        PauseLike(MCD_PAUSE).setDelay(16 hours);
+
+        // ---------- Emergency Shutdown Module Minimum Threshold Update ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+        // Increase the ESM (Emergency Shutdown Module) minimum threshold by 150k MKR from 150k MKR to 300k MKR
+        DssExecLib.setValue(MCD_ESM, "min", 300 * THOUSAND * WAD);
+
+        // ---------- Add LitePSM keeper network job ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+
+        // sequencer.addJob(0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be)
+        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_LITE_PSM_JOB);
```

Comment:
As per the [checklist rule](https://github.com/makerdao/pe-checklists/blob/e2376d43fe55e606669c48b44c41b0b8e7de70cc/spell/spell-crafter-mainnet-workflow.md?plain=1#L75), please add missing instructions with `// Note:` comment explaining that they were already taken

Reply:
Yeah, this is a weird case. The parameters are set in the constructor of the job and are immutable, meaning the spell itself won't touch it.
At first, I asked governance facilitators to remove it, however they are part of the parameters specified by BA Labs and the spell should ensure they are correct.

Reply:
It's fine to keep it in the sheet even if they are not directly part of the spell. The way it is now works – it makes clear for an external reviewer where to check them in case a confusion arises